components/intuiface/actions/available-experiences/available-experiences.mjs

```diff
@@ -0,0 +1,71 @@
+import app from "../../intuiface.app.mjs";
+
+export default {
+  key: "intuiface-available-experiences",
+  name: "Available Experiences",
```

Comment:
I think "List Available Experiences" would be more appropriate for the action name